docs/detections/alert-suppression.asciidoc

```diff
@@ -9,33 +9,42 @@ Alert suppression requires a https://www.elastic.co/pricing[Platinum or higher s
 preview::[]
 --
 
-Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a <<create-custom-rule,custom query>> detection rule. 
+Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by these detection rule types:
 
-Normally, when a rule matches multiple source events, it creates multiple alerts, one for each event. When alert suppression is configured, matching events are grouped by their values in a specified field, and only one alert is created for each group. You can configure alert suppression to create alerts each time the rule runs, or once within a specified time window. You can also specify multiple fields to group events by unique combinations of values.
+* <<create-custom-rule,Custom query>>
+* <<create-threshold-rule,Threshold>>
+
+Normally, when a rule matches multiple source events, it creates multiple alerts, one for each event. When alert suppression is configured, matching events are grouped by their values in a specified field, and only one alert is created for each group. Depending on the rule type, you can configure alert suppression to create alerts each time the rule runs, or once within a specified time window. You can also specify multiple fields to group events by unique combinations of values.
 
 The {security-app} displays several indicators in the Alerts table and the alert details flyout when a detection alert is created with alert suppression enabled. You can view the original events associated with suppressed alerts by investigating the alert in Timeline.
 
 NOTE: Alert suppression is not available for Elastic prebuilt rules. However, if you want to suppress alerts for a prebuilt rule, you can duplicate it, then configure alert suppression on the duplicated rule.
 
 === Configure alert suppression
 
-You can configure alert suppression when you create or edit a custom query rule. Refer to <<create-custom-rule>> for detailed instructions.
+You can configure alert suppression when you create or edit a supported rule type. Refer to <<create-custom-rule>> or <<create-threshold-rule>> for detailed instructions.
 
-. When configuring the rule type (the *Define rule* step for a new rule, or the *Definition* tab for an existing rule), enter one or more field names in *Suppress alerts by*.
+. When configuring the rule type (the *Define rule* step for a new rule, or the *Definition* tab for an existing rule), enter 1–3 field names to use for suppressing alerts. The setting where you enter the field names depends on the rule type:
```

Comment:
threshold rule also can use suppression without any selected **Group by** field

Reply:
@vitaliidm I don't understand how this would work. If the user doesn't specify a field in **Group by**, how will the system know which fields/values to suppress by?

Reply:
it only works for threshold rules. 
Threshold rule can be configured w/o Group by fields. So, suppression in this case works w/o Group by.

When there is no Group by fields, threshold rule just counts all events. So essentially, we may call it "suppression by the whole document"

https://github.com/elastic/security-docs/assets/92328789/18a34bc0-8a5d-4e51-8b31-52edf483e264

Reply:
OK, thanks for explaining! I pushed a commit that revised a bit to include leaving "Group by" empty: https://github.com/elastic/security-docs/pull/4467/commits/7726b3b14c20ad7475db5da23269e2ee237b3307